src/firefly/js/templates/lightcurve/LcUtil.jsx

```diff
@@ -16,6 +16,23 @@ export function getTypeData(key, val='', tip = '', labelV='', labelW) {
     };
 }
 
+/**
+ * Returns only numerical column names form raw lc table
+ * @param {Object} rawTbl
+ * @returns {TableColumn[]} - array of table columns objects
+ */
+export function getOnlyNumericalCol(rawTbl) {
+
+    var colType = ['double', 'd', 'long', 'l', 'int', 'i', 'float', 'f'];
+    return get(rawTbl, ['tableData', 'columns']).reduce((prev, col) => {
+        if ((colType.includes(col.type)) &&
+            (!has(col, 'visibility') || get(col, 'visibility') !== 'hidden')) {
+            prev.push(col.name);
+        }
+        return prev;
```

Comment:
This method was in WiseMissionOption.  I moved it here without making any changes.  I will make the necessary changes and then put it in TableUtils.js.